smartrent/badmagic
# i51: Deep Linking

This PR adds support for copying a deep link to a request.

Closes #51

## Steps to test

1. Set up a request however you like (with a body/qsParams/etc...)
2. Click "Copy Link"
3. Paste that link in another browser
4. Observe you were "deep linked" to that request

## Video Demo

https://user-images.githubusercontent.com/51095001/194618143-de0805fb-d594-47eb-8885-ed8333706e6a.mov
